terraform.tfvars.example

```diff
@@ -0,0 +1,6 @@
+env                 = ""
```

Comment:
Given that Terraform needs to be run from the `deploy_infrastructure.sh` script, I think it would be better to remove this from the vars example file and pass it to `terraform apply` directly. This would ensure the state bucket folder has the same name as the project name.